src/components/aside-bar/AsideBar.jsx

```diff
@@ -63,9 +71,13 @@ const AsideBar = () => {
               <label htmlFor="pants">
                 <input
                   type="checkbox"
+                  checked={pants}
                   id="pants"
                   onClick={(event) =>
-                    productPageDispatch({ type: "PANTS", payload: event.target.checked })
+                    productPageDispatch({
+                      type: "PANTS",
+                      payload: event.target.checked,
+                    })
```

Comment:
Suggestion: you can create re-usable component for this, as you are repeating same code and changing values.

Reply:
i will do it later